tests/lvm_dbus_tests.py

```diff
@@ -1201,6 +1202,10 @@ def _clean_up(self):
         except:
             pass
 
+        # lets help udev with removing stale symlinks
+        if not BlockDev.lvm_lvs("testVG") and os.path.exists("/dev/testVG/testCache_meta"):
+            shutil.rmtree("/dev/testVG")
```

Comment:
Isn't this racy with udev? Does this throw an exception when the path is not found?

Reply:
Good point, fixed.